app/src/pages/Register.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
```

Comment:
unclear on what this component is @martyboggs @tylerthome - do we need this or can we delete it?